Source/WebInspectorUI/UserInterface/Views/ScreenshotsTimelineView.css

```diff
@@ -24,16 +24,23 @@
  */
 
 .timeline-view.screenshots {
+    text-align: center;
     white-space: nowrap; 
     overflow-x: auto;
     scroll-snap-type: x mandatory;
 }
 
+.timeline-view.screenshots > .spacer {
+    display: inline-block;
+    height: 50%;
+}
```

Comment:
Nit: I wonder if flexbox layout would let us achieve the same result without the extra DOM.